react/components/Input/index.js

```diff
@@ -182,7 +183,7 @@ class Input extends Component {
     }
 
     return (
-      <label className="vtex-input w-100">
+      <label className="vtex-input w-100" data-testid={testId}>
```

Comment:
I think this should be inserted on the input itself.

Reply:
Thought so too but i think it's best at the root level. Developer can then use something like ```
cy.get('[data-testid=string]').within(result => {})
```
To look for label, input etc.

Reply:
If you are looking for a label you don't need the id, the id is best used to get the value of the input or if the input is disabled...

Reply:
You can still do a `cy.get('[data-testid=string] input')` and get that. You could also do a `cy.get('[data-testid=string] label')` to get the text for some reason.

Reply:
Updated. Does not make much sense to target other elements.